bitnami/airflow/README.md

```diff
@@ -0,0 +1,191 @@
+| `externalDatabase.database`               | External PostgreSQL database name                                                           | `nil`                                                        |
+| `externalDatabase.port`                   | External PostgreSQL port                                                                    | `nil`                                                        |
+| `redis.enabled`                           | Switch to enable or disable the Redis helm chart                                            | `true`                                                       |
+| `externalRedis.host`                      | External Redis host                                                                         | `nil`                                                        |
+| `externalRedis.port`                      | External Redis port                                                                         | `nil`                                                        |
+| `externalRedis.password`                  | External Redis password                                                                     | `nil`                                                        |
+| `metrics.enabled`                         | Start a side-car prometheus exporter                                                        | `false`                                                      |
+| `metrics.image.registry`                  | Airflow exporter image registry                                                             | `docker.io`                                                  |
+| `metrics.image.repository`                | Airflow exporter image name                                                                 | `pbweb/airflow-prometheus-exporter`                          |
+| `metrics.image.tag`                       | Airflow exporter image tag                                                                  | `latest`                                                     |
+| `metrics.image.pullPolicy`                | Image pull policy                                                                           | `IfNotPresent`                                               |
+| `metrics.image.pullSecrets`               | Specify docker-registry secret names as an array                                            | `[]` (does not add image pull secrets to deployed pods)      |
+| `metrics.podAnnotations`                  | Additional annotations for Metrics exporter                                                 | `{prometheus.io/scrape: "true", prometheus.io/port: "9112"}` |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
```

Comment:
I do not see any mention to the fernet, which is required when using the values-production.yaml file

Reply:
added a small section